pkg/solana/chainreader/account_read_binding.go

```diff
@@ -15,6 +16,8 @@ type BinaryDataReader interface {
 	ReadAll(context.Context, solana.PublicKey) ([]byte, error)
 }
 
+const maxResultAge = time.Minute
+
```

Comment:
what's this for and can we either remove/document?

Reply:
Shoot, that needs to be removed.